src/test/java/ancaMarian/tests/tema6/HerokuAlerts.java

```diff
@@ -0,0 +1,114 @@
+package ancaMarian.tests.tema6;
+
+import ancaMarian.driver.BrowserManager;
+import ancaMarian.utils.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.io.File;
+import java.io.IOException;
+
+
+public class HerokuAlerts {
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws InterruptedException {
+    navigateToHerokuHomePage();
+    maximizeWindow(driver);
+    openAlertsPage();
+    openFirstAlert();
+    acceptFirstAlert();
+    openSecondAlert();
+    acceptAlert();
+    openThirdAlert();
+    completeThirdAlert();
+    closeBrowser();
+
+    navigateToHerokuHomePage();
+    try {
+        openSecondAlert();
+    } catch (NoSuchElementException e){
+        FileUtils.takeScreenshot(driver, "alertNotFound");
+        System.out.println("Nu s-a gasit elementul pe pagina. S-a salvat un screenshot.");
+        } finally {
+        closeBrowser();}
+    }
+
+    public static void navigateToHerokuHomePage(){
+        driver = BrowserManager.createChromeDriver();
```

Comment:
Replace BrowserManager.createChromeDriver() with BrowserManager.createDriver() - the method from your package. Thanks!